includes/Setup.php

```diff
@@ -176,10 +176,9 @@ private function registerI18n() {
 		$smwgIP = $this->applicationFactory->getSettings()->get( 'smwgIP' );
 
 		$this->globalVars['wgMessagesDirs']['SemanticMediaWiki'] = $smwgIP . 'i18n';
-		$this->globalVars['wgExtensionMessagesFiles']['SemanticMediaWiki'] = $smwgIP . 'languages/SMW_Messages.php';
```

Comment:
I wonder why this worked before since there is no "SMW_Messages.php", i.e. shim. Hmm ...